packages/asc-ui/src/components/FilterTag/FilterTag.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+
+import { Close } from '@datapunt/asc-assets'
+import Tag, { Props as TagProps } from '../Tag/Tag'
+import FilterTagStyle from './FilterTagStyle'
+import Icon from '../Icon'
+
+export interface FilterTagProps extends TagProps {}
+
+const FilterTag: React.FC<FilterTagProps> = ({
+  filterTagOnClick,
```

Comment:
Would just call this `onClick` as that's how it's done for other components in the project as well